lib/src/main/kotlin/com/lemonappdev/konsist/core/ext/PathExt.kt

```diff
@@ -1,10 +1,11 @@
 package com.lemonappdev.konsist.core.ext
 
+import com.lemonappdev.konsist.core.util.PathHelper.toMacOsSeparator
+import com.lemonappdev.konsist.core.util.PathHelper.toOsSeparator
 import java.io.File
 
 val sep: String = File.separator
 
-fun String.toOsSeparator(): String = replace("/", File.separator)
-    .replace("\\", File.separator)
+internal fun String.toOsSeparator(): String = toOsSeparator(this)
```

Comment:
you don't need these extensions - why not use `PathHelper` directly everywhere where these redundant extensions are used?